sigmastate/src/test/scala/sigmastate/helpers/SigmaPPrint.scala

```diff
@@ -60,24 +72,90 @@ object SigmaPPrint extends PPrinter {
       Tree.Literal("SBooleanArray")
     case SPair(l, r) =>
       Tree.Apply("SPair", treeifySeq(Array(l, r)))
+    case t: PrimitiveType[_] =>
+      Tree.Literal(s"RType.${t.name}Type")
   }
 
   val exceptionHandlers: PartialFunction[Any, Tree] = {
-    case ex: ArithmeticException =>
-      Tree.Apply("new ArithmeticException", treeifySeq(Seq(ex.getMessage)))
+    case ex: Exception =>
+      Tree.Apply(s"new ${ex.getClass.getSimpleName}", treeifySeq(Seq(ex.getMessage)))
+  }
+
+  def treeifyByteArray(bytes: Array[Byte]): Tree = {
```

Comment:
If this class will be moved out of testing scope (I guess it will eventually), please provide ScalaDoc here and for other public methods of this class.

Reply:
made all methods either private or protected. also added ScalaDocs